<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>520</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_chatMembers">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>21</y>
     <width>271</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_conversatie">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>671</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Input">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
